src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -736,86 +720,92 @@ public String run(final Matcher m) {
 		}
 		return loadedFiles;
 	}
-	
+
 	/**
 	 * Loads structure of given script, currently only for functions. Must be called before
 	 * actually loading that script.
+	 *
 	 * @param f Script file.
 	 */
 	@SuppressWarnings("resource") // Stream is closed in Config constructor called in loadStructure
-	public final static @Nullable Config loadStructure(final File f) {
+	public static @Nullable
+	Config loadStructure(final File f) {
 		if (!f.exists()) { // If file does not exist...
 			unloadScript(f); // ... it might be good idea to unload it now
 			return null;
 		}
-		
+
 		try {
 			String name = Skript.getInstance().getDataFolder().toPath().resolve(Skript.SCRIPTSFOLDER).relativize(f.toPath()).toString();
 			assert name != null;
 			return loadStructure(new FileInputStream(f), name);
 		} catch (final IOException e) {
 			Skript.error("Could not load " + f.getName() + ": " + ExceptionUtils.toString(e));
 		}
-		
+
 		return null;
 	}
-	
+
 	/**
 	 * Loads structure of given script, currently only for functions. Must be called before
 	 * actually loading that script.
+	 *
 	 * @param source Source input stream.
-	 * @param name Name of source "file".
+	 * @param name   Name of source "file".

```

Comment:
One space after parameter name.

Reply:
IntelliJ tries to align this to columns automatically. Do you think it's a bad practice? I wouldn't say so, Java itself does that, look for example here http://www.docjar.net/html/api/java/util/Collections.java.html
```
   62    * @author  Josh Bloch
   63    * @author  Neal Gafter
   64    * @see     Collection
   65    * @see     Set
   66    * @see     List
   67    * @see     Map
   68    * @since   1.2
```
I'm pretty sure Eclipse should have some option for this too

Reply:
I mean, javadocs are theoretically meant to probably mainly be used with a website, but I personally (and I don't think I stand out much here) 98% of the time just look at them along with the code in the IDE, either source or decompiled, and this improves readability greatly, and is automated, so I'd say it's good